### PR TITLE
add german translation

### PR DESCRIPTION
**Pull request by RobertRR11**

### CODE REVIEW

**Comment by Rojer-GH** [comment]
As a german citizen, I would suggest some changes:

        "arl.misc.shift_for_info": "&7Halte &bSHIFT&7 für mehr Info",
	"arl.misc.right_click_add": "Rechtsklick zum Hinzufügen"

**Comment by RobertRR11** [comment]
Alright, I changed those things.

File: src/main/resources/assets/autoreglib/lang/de_de.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"arl.misc.shift_for_info": "&7Halte &bSHIFT&7 für mehr Info",
+	"arl.misc.right_click_add": "Rechtsklick zum Hinzüfügen"
```
Comment (EltrutLikes) [inline src/main/resources/assets/autoreglib/lang/de_de.json:3]: The word is "Hinz**u**fügen", not "Hinz**ü**fügen"

**Review by EltrutLikes — COMMENTED** [review]
I know this PR is old, but as the main German translator for the Quark mod, I left a comment regarding the spelling of another one of the words (sorry)

File: src/main/resources/assets/autoreglib/lang/de_de.json (hunk 1 above)
Comment (RobertRR11) [reply src/main/resources/assets/autoreglib/lang/de_de.json:3]: oh my god how did I do so many things wrong, something like this never happened to me

**Pull request closed without merging**